lib/pelemay.ex

```diff
@@ -171,10 +179,10 @@ defmodule Optimizer.Enum do
 
     func_name = func_name |> String.to_atom()
 
-    quote do: unquote(module).unquote(func_name)
+    quote do: Temp.unquote(func_name)
```

Comment:
I think `Temp` is not suitable name.
We should adopt an alternative name after `Module`.